concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1942,6 +1942,23 @@ jobs:
                 - -e
                 - -c
                 - |
+                  # FIXME: Remove this once
+                  # https://github.com/cloudfoundry/cf-acceptance-tests/pull/188 has been merged
```

Comment:
There is not, but I can add it. Anyway, that code will fail if you change the version of cf-release